DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Operations/Send/RootObjectBuilder.cs

```diff
@@ -26,12 +26,6 @@ public RootObjectBuilder(IUnitOfWorkFactory unitOfWorkFactory)
     CancellationToken ct = default
   )
   {
-    if (!objects.Any())
```

Comment:
Why wasn't this required? was this check already happening somewhere else?

Reply:
yep, it moved to the send function, we throw right away